KT-70 legacy instrument

i'm seeing the following lines in my fgfs log i don't know the details to fix this but i have seen it mentioned somewhere... maybe not this instrument but a similar "legacy instrument" message...

```
cockpit:3:/home/myuser/flightgear-dev/next/flightgear/src/Instrumentation/instrument_mgr.cxx:76:Reading instruments from /home/myuser/myflightgear/Aircraft/c172p-detailed/Systems/instrumentation.xml
instruments:4:/home/myuser/flightgear-dev/next/flightgear/src/Instrumentation/heading_indicator_dg.cxx:42:Error in DG heading-indicator config logic
instruments:4:/home/myuser/flightgear-dev/next/flightgear/src/Instrumentation/heading_indicator_dg.cxx:44:Section = heading-indicator
instruments:4:/home/myuser/flightgear-dev/next/flightgear/src/Instrumentation/instrument_mgr.cxx:185:KT-70 legacy instrument compatibility. Please update aircraft to use transponder directly
```

--- a/Nasal/engine.nas
+++ b/Nasal/engine.nas
@@ -89,7 +89,9 @@ var update = func {
     # We use the mixture to control the engines, so set the mixture
     var usePrimer = getprop("/controls/engines/engine/use-primer") or 0;
 
-    if (outOfFuel) {
+    var engine_running = getprop("/engines/engine/running");
+
+    if (outOfFuel and (engine_running or usePrimer)) {
         print("Out of fuel!");
         gui.popupTip("Out of fuel!");
     }

--- a/Systems/instrumentation.xml
+++ b/Systems/instrumentation.xml
@@ -74,16 +74,11 @@ file, these values will be used (they are hardcoded).
     <number>0</number>
   </marker-beacon>
 
-  <heading-indicator-dg>
+  <heading-indicator>
     <name>heading-indicator</name>
     <number>0</number>
     <suction>/systems/vacuum/suction-inhg</suction>
-  </heading-indicator-dg>
-
-  <KT-70>
-    <name>kt-70</name>
-    <number>0</number>
-  </KT-70>
+  </heading-indicator>
 
   <magnetic-compass>
     <name>magnetic-compass</name>

--- a/gui/dialogs/c172p-menu.xml
+++ b/gui/dialogs/c172p-menu.xml
@@ -32,7 +32,7 @@
             </item>
 
             <item>
-                <name>Autostart</name>
+                <label>Autostart</label>
                 <binding>
                     <command>nasal</command>
                     <script>c172p.autostart()</script>

